cli/cache/fast_check.rs

```diff
@@ -81,13 +84,14 @@ impl FastCheckCacheInner {
       WHERE
         hash=?1
       LIMIT 1";
-    let res = self
-      .conn
-      // key is a string because SQLite can't handle u64
-      .query_row(query, params![key.as_u64().to_string()], |row| {
+    let res = self.conn.query_row(
+      query,
+      params![CacheDBHash::new(key.as_u64())],
```

Comment:
Opened https://github.com/denoland/deno_graph/pull/491 to make the hash have the deno_graph version in it.